### ISSUE TITLE
Guilherme_Pereira_2019: Processing submission

### ISSUE THREAD

**Issue by ehwenk** [traitecoevo/austraits.build#283]
**Source**: Caio Guilherme Pereira et al (2019) Trait convergence in photosynthetic nutrient‐use efficiency along a 2‐million year dune chronosequence in a global biodiversity hotspot *Journal of Ecology* http://doi.org/10.1111/1365-2745.13158
**Custodian**: Caio Guilherme Pereira <caiogp@mit.edu>
**Handled by**: ?Austraits team member?

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [x] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [x] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [x] Metadata complete, builds into Austraits
- [x] Report reviewed by AusTraits team
- [x] Report sent to contributor ( label as `waiting on response` )
- [x] Response received from contributor
- [x] Files updated & report finalised
- [x] Final checks ( all tests pass )
- [x] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
